srcpkgs/lemurs/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'lemurs'
+pkgname=lemurs
+version=0.4.0
+revision=1
+archs="x86_64*"
```

Comment:
Is this actually limited to `x86_64*`?

Reply:
No, this is just me targeting my own computer.
I did not want to have false hope for other people on other architectures I guess.

I will not have time to make it work in the next weeks, if you want to take over I would advise you to start from @tranzystorekk branch rather than mine (see comments above)!